### PR TITLE
Hammingloss

### LINKED ISSUE #1979: Tutorial for Hamming loss
I recently used Hamming loss from tfa for my thesis project. If it sounds useful, I'd like to add a tutorial on how to use the metric and probably other metrics in general. It took me some time to get around it so maybe this could save some time for others. Do let me know if this sounds like a worthwhile addition

- Are you willing to contribute it: Yes
- Are you willing to maintain it going forward?: In the near future, yes
- Is there a relevant academic paper?: No
- Is there already an implementation in another framework?: Requesting a tutorial 
- Was it part of tf.contrib?: No

**Which API type would this fall under - Metrics

People who are working on multi-label classification problems would benefit from such a tutorial as tfa is the only library I could find for using these metrics while model training in tensorflow/keras.

P.S - I haven't done many open-source contributions and I'm still learning

### PR DESCRIPTION
**Pull request by harshithere**
Added a tutorial for usage of Hamming loss for multilabel classification
Issue number: Fixes #1979 

Colab notebook: https://colab.research.google.com/github/harshithere/addons/blob/hammingloss/docs/tutorials/metrics_hammingloss.ipynb

- [ ] Bug fix
- [x] New Tutorial
- [ ] Updated or additional documentation
- [ ] Additional Testing
- [ ] New Activation (Requires creating an issue first)
    - [ ] The changes conform to the [contribution guidelines](https://github.com/tensorflow/addons/blob/master/tensorflow_addons/activations/README.md#contribution-guidelines)
- [ ] New Callback (Requires creating an issue first)
    - [ ] The changes conform to the [contribution guidelines](https://github.com/tensorflow/addons/blob/master/tensorflow_addons/callbacks/README.md#contribution-guidelines)
- [ ] New Image addition (Requires creating an issue first)
    - [ ] The changes conform to the [contribution guidelines](https://github.com/tensorflow/addons/blob/master/tensorflow_addons/image/README.md#contribution-guidelines)
- [ ] New Layer (Requires creating an issue first)
    - [ ] The changes conform to the [contribution guidelines](https://github.com/tensorflow/addons/blob/master/tensorflow_addons/layers/README.md#contribution-guidelines)
- [ ] New Loss (Requires creating an issue first)
    - [ ] The changes conform to the [contribution guidelines](https://github.com/tensorflow/addons/blob/master/tensorflow_addons/losses/README.md#contribution-guidelines)
- [ ] New Metric (Requires creating an issue first)
    - [ ] The changes conform to the [contribution guidelines](https://github.com/tensorflow/addons/blob/master/tensorflow_addons/metrics/README.md#contribution-guidelines)
- [ ] New Optimizer (Requires creating an issue first)
    - [ ] The changes conform to the [contribution guidelines](https://github.com/tensorflow/addons/blob/master/tensorflow_addons/optimizers/README.md#contribution-guidelines)
- [ ] New RNN Cell (Requires creating an issue first)
    - [ ] The changes conform to the [contribution guidelines](https://github.com/tensorflow/addons/blob/master/tensorflow_addons/rnn/README.md#contribution-guidelines)
- [ ] New Seq2seq addition (Requires creating an issue first)
    - [ ] The changes conform to the [contribution guidelines](https://github.com/tensorflow/addons/blob/master/tensorflow_addons/seq2seq/README.md#contribution-guidelines)
- [ ] New Text addition (Requires creating an issue first)
    - [ ] The changes conform to the [contribution guidelines](https://github.com/tensorflow/addons/blob/master/tensorflow_addons/text/README.md#contribution-guidelines)

# How Has This Been Tested?

If you're adding a bugfix or new feature please describe the tests that you ran to verify your changes:
*  

# Checklist:

- [x] I've properly [formatted my code according to the guidelines](https://github.com/tensorflow/addons/blob/master/CONTRIBUTING.md#coding-style)
    - [ ] By running Black + Flake8
    - [ ] By running pre-commit hooks
- [ ] This PR addresses an already submitted issue for TensorFlow Addons
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] This PR contains modifications to C++ custom-ops

### CODE REVIEW

**Comment by harshithere** [comment]
@googlebot I signed it!

**Comment by harshithere** [comment]
@WindQAQ I read in the guidelines that PRs should be without code output.
Do you mean to show the output in the colab notebook for reference?

**Comment by WindQAQ** [comment]
> @WindQAQ I read in the guidelines that PRs should be without code output.
> Do you mean to show the output in the colab notebook for reference?

I see. Let me check with other members about how's the policy going.

Hi @seanpmorgan, do you need to include cell outputs for tutorials? Seems that there's a sentence in the template:
>This notebook has the "Omit code cell output when saving this notebook" option set. GitHub refuses to diff notebooks with large diffs (inline images).

and tutorials in core TF do not include cell outputs as well.

https://github.com/tensorflow/docs/tree/master/site/en/tutorials
https://github.com/tensorflow/addons/blob/master/docs/tutorials/_template.ipynb

**Comment by seanpmorgan** [comment]
IMO having cell output is useful for reading the tutorials on the tensorflow webpage. Not all cell outputs are required though only the ones that are useful for the story you're trying to tell.

**Comment by WindQAQ** [comment]
> IMO having cell output is useful for reading the tutorials on the tensorflow webpage. Not all cell outputs are required though only the ones that are useful for the story you're trying to tell.

Agree +1

File: docs/tutorials/metrics_hammingloss.ipynb [hunk 1]
```diff
@@ -0,0 +1,276 @@
+{
```
Comment (WindQAQ) [inline docs/tutorials/metrics_hammingloss.ipynb:1]: <p>I do not see this button in our template. Is it added on your own or where do you can this?</p><p> </p><p><a href="https://github.com/tensorflow/addons/blob/master/docs/tutorials/_template.ipynb" target="_blank">https://github.com/tensorflow/addons/blob/master/docs/tutorials/_template.ipynb</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/addons/pull/2044/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

**Comment by harshithere** [comment]
@WindQAQ Thanks for the review :)
I've updated the _toc file. Is that all or are there any other changes required?

File: docs/tutorials/metrics_hammingloss.ipynb (hunk 1 above)
Comment (WindQAQ) [reply docs/tutorials/metrics_hammingloss.ipynb:1]: @harshithere here it is.

**Comment by seanpmorgan** [comment]
Thank you for your contribution. We sincerely apologize for any delay in reviewing, but TensorFlow Addons is transitioning to a minimal maintenance and release mode. New features will not be added to this repository. For more information, please see our public messaging on this decision:
[TensorFlow Addons Wind Down](https://github.com/tensorflow/addons/issues/2807)

Please consider sending feature requests / contributions to other repositories in the TF community with a similar charters to TFA:
[Keras](https://github.com/keras-team/keras)
[Keras-CV](https://github.com/keras-team/keras-cv)
[Keras-NLP](https://github.com/keras-team/keras-nlp)
